server/mdm/apple/commander.go

```diff
@@ -136,6 +144,33 @@ func (svc *MDMAppleCommander) InstallEnterpriseApplication(ctx context.Context,
 	return svc.EnqueueCommand(ctx, hostUUIDs, raw)
 }
 
+type installEnterpriseApplicationPayload struct {
+	Manifest    *appmanifest.Manifest
+	RequestType string
+}
+
+func (svc *MDMAppleCommander) InstallEnterpriseApplicationWithEmbeddedManifest(
+	ctx context.Context,
+	hostUUIDs []string,
+	uuid string,
```

Comment:
Nit: Should it just generate a new UUID every time? (I don't see why we'd want a command with a specific UUID other than a new random one)

Reply:
for some commander methods we need to control the uuid, so we made a convention of receiving the UUID for all, but maybe the time has come to revisit? For now I vote to keep it for consistency.

Reply:
Sure, yeah just a nit.